msbuild/Xamarin.MacDev.Tasks.Core/Tasks/UnpackLibraryResourcesTaskBase.cs

```diff
@@ -83,6 +89,15 @@ public override bool Execute ()
 			return !Log.HasLoggedErrors;
 		}
 
+		bool IsFrameworkAssembly (ITaskItem asm)
+		{
+			foreach (var dir in TargetFrameworkDirectory) {
+				if (asm.ItemSpec.StartsWith (dir.ItemSpec, StringComparison.Ordinal))
+					return true;
```

Comment:
Depending on how the TaskItem is created, the ItemSpec may be a relative path so it *may* be better to use the full path of each item using this approach:

```csharp
var path = item.GetMetadata ("FullPath");
```

Obviously we can do this outside the loop for `asm` as an optimization.

My other suggestion is that we may want to use an `OrdinalIgnoreCase` compare since the Mac file system is case insensitive by default.